### PR TITLE
ci(test): trigger on pull_request and master

### PR DESCRIPTION
**Pull request by steebchen**

### CODE REVIEW

**Comment by janpio** [comment]
Is there a way to not have 2 workflows running the tests if we ourselves open a Pull Requests for a branch in this repo?

**Comment by steebchen** [comment]
I don't think so, that's why I didn't merge yet

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -1,5 +1,12 @@
 name: test
-on: push
+on:
+  push:
+    branches:
+      - master
+      - alpha
+      - dev
+      - patch-dev
+  pull_request:
```
Comment (janpio) [inline .github/workflows/test.yaml:9]: is this correct with the colon but not sub value?

Comment (steebchen) [reply .github/workflows/test.yaml:9]: yep

Comment (steebchen) [reply .github/workflows/test.yaml:9]: seems a bit weird, but works indeed

**Pull request merged**